crowbar_framework/app/models/api/node.rb

```diff
@@ -15,6 +15,56 @@
 #
 
 module Api
-  class Node
+  class Node < Tableless
+    def repocheck(options = {})
+      addon = options.fetch(:addon, "os")
+      features = ["os"]
+      features.push(addon) if addon
+
+      {}.tap do |ret|
```

Comment:
the ret variable is somehow meaning less, maybe you can replace it with a better name